src/client/pages/create.js

```diff
@@ -223,13 +239,18 @@ class Create extends React.Component {
               <tr key={i}>
                 <td>{this.state.names[i]}</td>
                 <td>
-                  <a href={`${window.location.origin}/${this.state.gameID}/${i}/${this.state.secret[i]}`} target="_blank" rel="noopener noreferrer">{window.location.origin}/{this.state.gameID}/{i}/{this.state.secret[i]}</a>
+                  <a href={`${this.url(i)}`} target="_blank" rel="noopener noreferrer">{window.location.origin}/{this.state.gameID}/{i}/{this.state.secret[i]}</a>
+                </td>
+                <td>
+                  <CopyButton text={this.url(i)} />
```

Comment:
You don't pass the color that you marked as required in `<CopyButton/>`?

Reply:
I didn't mean to make it required, I just wanted it to be passable, I've fixed this